build/vsts_build.yaml

```diff
@@ -839,7 +841,9 @@ phases:
   condition: "and(succeeded(),eq(variables['RunApexTests'], 'true')) "
   queue:
     name: DDNuGet-Windows
-    demands: DotNetFramework
+    demands:
+    - DotNetFramework
+    - Agent.Name -equals DDNuGetW001
```

Comment:
I assume this is for debugging?

For the benefit of reviewers, you can avoid polluting this PR with debugging commits by creating a branch off this branch and pushing changes to that new branch.  Integrate changes back when appropriate.